### ISSUE TITLE
Preview 17 generate with custom path not working

### ISSUE THREAD

**Issue by mazzaker** [prisma/photonjs#309]
I get this error:
```
> Downloading debian-openssl-1.1.x binary for query-engine [====================] 100%

Generating Photon.js to ./generated/photon/photonAdmin
Error: Error: ENOENT: no such file or directory, copyfile '/c/Users/bjoer/Documents/Projects/necom/node_modules/prisma2/build/query-engine-debian-openssl-1.1.x' -> '/c/Users/bjoer/Documents/Projects/necom/admin/generated/photon/photonAdmin/runtime/query-engine-deb
ian-openssl-1.1.x'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

My schema:

```
datasource db {
    provider = "mysql"
    url      = env("MYSQL_URL")
    default  = true
}

generator photon {
    provider = "photonjs"
    output = "../generated/photon/photonAdmin"
}
...
```

**Comment by pantharshit00** [comment]
I think this might be a duplicate of https://github.com/prisma/lift/issues/219

**Comment by pantharshit00** [comment]
@mazzaker 

Can you please tell me which OS you are using? Your path seems to be either coming from windows git bash or windows subsystem for linux.

**Comment by janpio** [comment]
I would guess WSL, because it downloads the debian binary.

I tried to recreate this with a new project (via `prisma2 init`), but everything worked:

```
jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway$ npm install -g prisma2
/home/jan/.nvm/versions/node/v12.7.0/bin/prisma2 -> /home/jan/.nvm/versions/node/v12.7.0/lib/node_modules/prisma2/build/index.js

> prisma2@2.0.0-preview017.1 install /home/jan/.nvm/versions/node/v12.7.0/lib/node_modules/prisma2
> node download-build/index.js || echo ""

> Downloading debian-openssl-1.1.x binary for query-engine and migration-engine [====================] 100%



+ prisma2@2.0.0-preview017.1
added 1 package from 1 contributor in 24.413s
jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway$ prisma2 init 309

 SUCCESS  The 309 directory was created!
 SUCCESS  Prisma is connected to your database at file:dev.db

┌─ Next steps ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                                                                                                                                                                                                                                                                │
│   Navigate into the project directory:                                                                                                                                                                                                                                                         │
│   $ cd 309                                                                                                                                                                                                                                                                                     │
│                                                                                                                                                                                                                                                                                                │
│   Start Prisma's development mode to enable access to                                                                                                                                                                                                                                          │
│   Prisma Studio and watch schema.prisma for changes:                                                                                                                                                                                                                                           │
│   $ prisma2 dev                                                                                                                                                                                                                                                                                │
│                                                                                                                                                                                                                                                                                                │
│   Start the REST API server (in a new terminal window/tab):                                                                                                                                                                                                                                    │
│   $ npm run dev                                                                                                                                                                                                                                                                                │
│                                                                                                                                                                                                                                                                                                │
│   Learn more about the Prisma Framework:                                                                                                                                                                                                                                                       │
│   https://github.com/prisma/prisma2/                                                                                                                                                                                                                                                           │
│                                                                                                                                                                                                                                                                                                │
│   If you encounter any issues, please report them here:                                                                                                                                                                                                                                        │
│   https://github.com/prisma/prisma-examples/issues/new                                                                                                                                                                                                                                         │
│                                                                                                                                                                                                                                                                                                │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway$ cd 309
jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway/309$ prisma2 generate
> Downloading debian-openssl-1.1.x binary for query-engine [====================] 100%

Generating Photon.js to ./node_modules/@prisma/photon
Done in 2.28s
```

Also adding `output = "../generated/photon/photonAdmin"` to the generator, still worked:

```
jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway/309$ prisma2 generate
> Downloading debian-openssl-1.1.x binary for query-engine [====================] 100%

Generating Photon.js to ./generated/photon/photonAdmin
Done in 3.41s

jan@Surface-Book-2:/mnt/c/Users/Jan/Documents/throwaway/309$
```

**Comment by pantharshit00** [comment]
@janpio There is no mount folder `/mnt/` and C drive is accessible directly so there is a chance this is not wsl but git bash. So that is why I asked @mazzaker to confirm.

Also, I just confirmed a bug with exact same error message so this might be related and can be closed: https://github.com/prisma/prisma2/issues/993

**Comment by mazzaker** [comment]
Sorry for the late reply.
Yes I am using WSL 1 debian. After deleting the whole project and start only with sources this error was gone.
Thank you for your support

**Comment by tvvignesh** [comment]
@pantharshit00 I faced the same error today when I deleted the generated folder and tried to run prisma2 generate again. It was not able to find/generate the runtime folder which it used to generate before.

To fix this, I copied the runtime folder I had from another project generated few days ago and pasted it there, ran prisma2 generate and it worked. Screenshot below.

![Screenshot from 2019-12-26 00-40-46](https://user-images.githubusercontent.com/1165845/71449692-82120800-2778-11ea-8918-9fcba42c1b1a.png)

This is the config I have in my schema.prisma:

```
datasource postgres {
    provider = "postgres"
    url      = env("POSTGRES_URL")
    default  = true
    enabled  = true
}

generator photon {
    provider = "photonjs"
    output   = "../../../generated/photon"
}
```
